### PR TITLE
I18n fix

### PR DESCRIPTION
**Pull request by ydeshayes**
Hello !

https://github.com/institut-de-genomique/Ultimate-DataTable/issues/14

I add a new init function in the I18n service that select the language. We can now give a String or an array of String to the I18n service constructor in order to use the [navigator.languages](https://developer.mozilla.org/en-US/docs/Web/API/NavigatorLanguage/languages) new generation browser function.

The init function try to find a translation with the all string (xx-XX) and if there is no translation for the extended language, we try with the primary language only (xx). The default language is 'en'.

The Messages function is now pure, because the check of the language is only on the init function.

Some additionnal [informations](http://tools.ietf.org/html/rfc4646#section-2.2.4).

### CODE REVIEW

File: src/services/udt-i18n.js [hunk 1]
```diff
@@ -1,10 +1,50 @@
...
+              // We first try to find the entire language string
+              // Primary Language Subtag with Extended Language Subtags
+              if(this.tanslationExist(language)) {
+                this.preferedLanguage = language;
+                return true;
+              }
+
+              // Then we try with only Primary Language Subtag
+              var splitedLanguages = language.split('-');
+              if(splitedLanguages > 1) {
```
Comment (Jenselme) [inline src/services/udt-i18n.js]: Use `splitedLanguages.length > 1` here and it will be perfect.

Comment (ydeshayes) [inline src/services/udt-i18n.js]: Nice catch, I commit a fix.

**Comment by Jenselme** [comment]
LGTM. Can you merge this?

**Comment by Jenselme** [comment]
Thanks.

**Pull request merged**